### PR TITLE
added Dockerfile

### PR DESCRIPTION
**Pull request by FexbYk23**

### CODE REVIEW

**Review by doyu — COMMENTED** [review]
Looks OK but I'd like to know some points before pulling in.

Probably next step would be:
- Run unit tests if specificed docker run parameter
- Run Hello world if specificed docker run parameter
- Add acceptance test (e.g. Robot Framework?) if specified docker run parameter
- Add optional execution of '-gcov' coverage if specified docker run parameter

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM ubuntu
+
+RUN apt update && apt install -y git make unzip curl g++ python3 pip wget
+RUN pip3 install Pillow Wave
```
[the remark this answers is not in the document]
Comment (doyu) [reply Dockerfile:4]: You may want to refer to some other Dockerfile for python where they may use "requirements.txt" to collecte pacakges. Then you won't need to update this Docker file.

[the remark this answers is not in the document]
Comment (FexbYk23) [reply Dockerfile]: Pillow is an [image library](https://pypi.org/project/Pillow/). It's required by [tflite-micro/tensorflow/lite/micro/tools/generate_cc_arrays.py](https://github.com/tensorflow/tflite-micro/blob/0c425da368ff3b4ce4feda37a3de08a0f58c5d06/tensorflow/lite/micro/tools/generate_cc_arrays.py#L26) which runs during make.

Wave appears to be some kind of mistake by me, as the Dockerfile works fine without it. I've now removed it.

**Review by doyu — COMMENTED** [review]
Looks good to me. Thanks!

**Pull request merged**